src/Modules/OrchardCore.Commerce.Payment/Services/CheckoutAddressService.cs

```diff
@@ -0,0 +1,23 @@
+using OrchardCore.Commerce.Payment.Settings;
+using OrchardCore.Commerce.Payment.ViewModels;
+using OrchardCore.Settings;
+using System.Threading.Tasks;
+
+namespace OrchardCore.Commerce.Payment.Services;
+
+public class CheckoutAddressService : ICheckoutAddressService
+{
+    private readonly ISiteService _siteService;
+
+    public CheckoutAddressService(ISiteService siteService) => _siteService = siteService;
+
+    public virtual async Task<bool> ShouldIgnoreAddressAsync(CheckoutViewModel checkoutViewModel)
+    {
+        var settings = (await _siteService.GetSiteSettingsAsync()).As<CheckoutAddressSettings>();
+        var shouldIgnore = settings.ShouldIgnoreAddress;
+
+        var total = checkoutViewModel.SingleCurrencyTotal.Value;
```

Comment:
I don't think that empty line is helpful here.
```suggestion
        var total = checkoutViewModel.SingleCurrencyTotal.Value;
```